Cargo.toml

```
@@ -57,6 +57,9 @@ version = "=0.5.4"
 [dependencies.snarkvm-utilities]
 version = "=0.5.4"
 
+[dependencies.snarkvm-parameters]
+version = "0.5.4"
```

```suggestion
version = "=0.5.4"
```
so that we can periodically do `cargo update` even if there's a been a snarkVM update that we haven't accounted for yet

---

in general, this is a snarkvm issue with not following semver, but because of that, sounds good. will change.